src/test/java/com/openlattice/chronicle/ChronicleServerTests.java

```diff
@@ -1,11 +1,497 @@
+            .of( "com.android.chrome", "Chrome" );
+    private static final Pair<String, String>         MAPS                      = Pair
+            .of( "com.google.android.apps.maps", "Maps" );
+    private static final Pair<String, String>         PHONE                     = Pair
+            .of( "com.android.dialer", "Phone" );
+
+    private static       Map<String, UUID>            entitySetNameIdMap        = new HashMap<>();
+    private static       UUID                         fullNamePTID;
+    private static       UUID                         durationPTID;
+    private static       UUID                         titlePTID;
+    private static       UUID                         startDateTimePTID;
+    private static       UUID                         dateLoggedPTID;
+    private static UUID recordTypePTID;
+
+    private static DataApi       dataApi;
+    private static EdmApi        edmApi;
+    private static SearchApi     searchApi;
+    private static EntitySetsApi entitySetsApi;
+
+    ChronicleConfiguration configuration = new ChronicleConfiguration( "user", "password" );
+    @Inject
+    private EventBus eventBus;
+    ChronicleService chronicleService = new ChronicleServiceImpl( eventBus, configuration );
+
+    public ChronicleServerTests() throws ExecutionException {
+
+    }
+
+    @BeforeClass
+    public static void chronicleServerTest() throws Exception {
+
+        // replace this with an actual auth0_id token
+        String token = "";
+
+        LoadingCache<Class<?>, ApiClient> apiClientCache = CacheBuilder
+                .newBuilder()
+                .expireAfterWrite( Duration.ofHours( 10 ) )
+                .build( new CacheLoader<Class<?>, ApiClient>() {
+                    @Override
+                    public ApiClient load( Class<?> key ) throws Exception {
+                        // String token = MissionControl.getIdToken( TEST_USER, TEST_PASSWORD );
+                        return new ApiClient( RetrofitFactory.Environment.TESTING, () -> token );
+                    }
+                } );
+        ApiClient apiClient = apiClientCache.get( ApiClient.class );
+
+        dataApi = apiClient.getDataApi();
+        edmApi = apiClient.getEdmApi();
+        entitySetsApi = apiClient.getEntitySetsApi();
+        searchApi = apiClient.getSearchApi();
+
+        setUpTestingEnvironment();
+    }
+
+    private static void deleteEntities() {
+
+        Set<UUID> entitySetsIds = new HashSet<>();
+        entitySetsIds.add( entitySetNameIdMap.get( CHRONICLE_USER_APPS ) );
+        entitySetsIds.add( entitySetNameIdMap.get( APP_DATA_ES_NAME ) );
+        // entitySetsIds.add( entitySetNameIdMap.get( RECORDED_BY_ES_NAME ) );
+        // entitySetsIds.add( entitySetNameIdMap.get( USED_BY_ES_NAME ) );
+
+        for ( UUID entitySetId : entitySetsIds ) {
+            dataApi.deleteAllEntitiesFromEntitySet( entitySetId, DeleteType.Hard );
+        }
+    }
```

Comment:
This looks like a cleanup task since you are calling it in most of the tests. The `@Before` annotation exists for exactly this reason; `@Before` is executed before each `@Test`.

https://junit.org/junit4/javadoc/latest/index.html

Reply:
Dropping tests for now.